types/react-content-loader/index.d.ts

```diff
@@ -1,22 +1,44 @@
-// Type definitions for react-content-loader 2.0
+// Type definitions for react-content-loader 3.1
 // Project: https://github.com/danilowoz/react-content-loader
 // Definitions by: Alaa Masoud <https://github.com/alaatm>
+//                 Sam Walsh <https://github.com/samwalshnz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.6
 
-import * as React from 'react';
+import * as React from "react";
 
 export interface ContentLoaderProps {
     style?: React.CSSProperties;
-    type?: 'facebook' | 'instagram' | 'list' | 'bullet-list' | 'code';
+    type?: "facebook" | "instagram" | "list" | "bullet-list" | "code";
```

Comment:
Why double quotes?

Reply:
Apologies - it looks like my code formatter has taken over